### DISCUSSION TITLE
Hardware for Martin

### DISCUSSION THREAD

**Discussion by luggesexe** [maplibre/martin#1572] [Q&A] [answered]
Hello,

I am trying to serve "ready" vector tiles that I created with tilemaker with Martin. From my understanding the .mbtiles file that I am trying to serve is under the hood just a SQL Database. 

Apart from the storage space for the mbtiles file, what hardware requirements do I have to consider to serve it? Are there relevant known CPU/RAM indicators? 

Kindly!

**Comment by CommanderStorm** [comment, marked as answer]
Really depends on
- your users,
- your tiles (both area and what details),
- your indexes/file format used,
- your hardware, 
- extra dynamic data,
- ... 

I don't think any answer that you would get by this question would be applicable to *YOUR* concrete usecase and be way to general.
If I were you, I would just benchmark what *YOUR* usecase needs.

As a rule of thumb, we are pretty dang efficient..
So CPU and ram will only be used to a lower extent
